ya-runtime-erigon/src/main.rs

```diff
@@ -56,12 +60,19 @@ pub struct ErigonRuntime {
     erigon_pid: Option<Child>,
     rpcdaemon_pid: Option<Child>,
     erigon_password: Option<String>,
+    erigon_chain_id: Option<String>,
 }
 
 impl Runtime for ErigonRuntime {
     const MODE: RuntimeMode = RuntimeMode::Server;
 
     fn deploy<'a>(&mut self, ctx: &mut Context<Self>) -> OutputResponse<'a> {
+        let data_dir = ctx.conf.data_dir.clone();
+        let chain_subdirs: Vec<PathBuf> = SUPPORTED_CHAINS
+            .into_iter()
```

Comment:
If we use an enum type for chains/networks we use here `IterVariantNames` macro from [enum_derive](https://docs.rs/enum_derive/0.1.7/enum_derive/) crate.